frontend-ui/src/router/index.ts

```diff
@@ -1,26 +1,87 @@
 import PipelineView from '@/views/PipelineView.vue'
 import ScheduleDetailView from '@/views/ScheduleDetailView.vue'
+import SignInView from '@/views/SignInView.vue'
 import TaskDetailView from '@/views/TaskDetailView.vue'
 import { createRouter, createWebHistory } from "vue-router"
 
 const routes = [
     {
         path: '/',
         name: 'home',
-        component: PipelineView,
+        redirect: { name: 'pipeline' },
     },
     {
-      path: '/pipeline/:id',
+      path: '/pipeline',
+      name: 'pipeline',
+      component: PipelineView,
+    },
+    {
+      path: '/support-us',
+      name: 'support',
+      component: PipelineView,
+    },
+    {
+      path: '/sign-in',
+      name: 'sign-in',
+      component: SignInView,
+    },
+    {
+      path: '/change-password',
+      name: 'change-password',
+      component: TaskDetailView,
```

Comment:
I'm pretty sure you already now this, but this and all users paths will have to be fixed 🤣

Reply:
yeah, router was complaining about missing paths. 😅